authentication/authentication-styles-2.css

```diff
@@ -0,0 +1,110 @@
+body {
+    width: 100vw;
+    height: 100vh;
+    background-color: #EEEEEE;
+
+    padding: 20px;
+
+    display: flex;
+    flex-direction: column;
+    justify-content: center;
+    align-items: center;
+}
+
+.card {
+    max-width: 50%;
+    background-color: white;
+    display: flex;
+    flex-direction: row;
+
+    border-radius: 16px;
+    box-shadow: 4px 4px 8px grey;
+
+}
+
+a {
```

Comment:
Consider using specific classname or id. We should not style the whole element as it could affect all the `<a/>` element of other pages